dp/cloud/go/services/dp/storage/amc_manager.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import "magma/dp/cloud/go/services/dp/storage/db"
+
+// AmcManager is supposed to be a library that will replace radio controller
+// it is not supposed to be a service but rather an interface to database
+// could be implemented in this file as separate struct or combined with cbsd manager
+// also its method are supposed to be used in transaction (they should start a new one)
+type AmcManager interface {
+	// GetState is equivalent to GetState grpc method
```

Comment:
I suggest putting a description here, otherwise once this is implemented there will be no grpc method to refer to, thus this comment will become useless (Unless SOMEONE changes it :P)

Reply:
I think it's just a tip for us, for when we'll be implementing them. New Go implementations of these functions might differ from Python originals, so they should be commented accordingly anyway.

Reply:
done